helm/pay-wallet-values-prod.yaml

```diff
@@ -83,6 +83,7 @@ microservice-chart:
     WALLET_SERVICE_READ_TIMEOUT: "10000"
     WALLET_SERVICE_CONNECTION_TIMEOUT: "10000"
     WALLET_EXPIRATION_QUEUE_NAME: "pagopa-p-itn-pay-wallet-expiration-queue"
+    AZURE_EVENTHUB_CONNECTION_STRING: "kafka:29092"
```

Comment:
AZURE_EVENTHUB_CONNECTION_STRING it is an envSecret in main and thus conflicts with with this branch

Reply:
ops!

Reply:
fixed here d90cfc1d69c2572c34ff107ea143afe88dc9f1d4